firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -29,29 +31,23 @@ def render(self, value, obj=None):
 class UserForeignKeyWidget(ForeignKeyWidget):
     """Import/export widget to find/create users from their email.
 
-    When Django cannot find a User with a matching email address,
-    it will create a new one if create=True, otherwise, will return dummy user.
+    If Django cannot find a User with a matching email address, will return dummy user.
```

Comment:
Is there a reason not to create a user for unknown email addresses that show up in the created_by/modified_by fields?

Reply:
I had made this option but we were not actually using it anywhere. Should I revert the change and set the migrations to create users? I had thought that any users that were not migrated (e.g. `jenkins_cli`) would be better handled with the dummy user, once it looks like an administrator.